telegram/ext/_picklepersistence.py

```diff
@@ -63,6 +63,9 @@ def _custom_reduction(cls: TelegramObj) -> Tuple[Callable, Tuple[Type[TelegramOb
     works as intended.
     """
     data = cls._get_attrs(include_private=True)  # pylint: disable=protected-access
+    # MappingProxyType is not pickable, so we convert it to a dict
```

Comment:
```suggestion
    # MappingProxyType is not pickleable, so we convert it to a dict
```

Reply:
The correct term seems to be "picklable" from https://docs.python.org/3/library/pickle.html#what-can-be-pickled-and-unpickled (last point)